packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec

```diff
@@ -0,0 +1,40 @@
+%global repo_orgname project-receptor
+%global repo_name satellite-receptor-installer
+
+%global role_orgname %{repo_orgname}
+%global role_name satellite_receptor_installer
+
+Name: ansiblerole-satellite-receptor-installer
+Summary: Packaging of the satellite_receptor_installer Ansible role
+Version: 0.5.0
+Release: 1%{?dist}
+License: GPLv3
+
+Source0: https://github.com/%{repo_orgname}/%{repo_name}/archive/v%{version}.tar.gz#/%{role_name}-%{version}.tar.gz
+Url: https://github.com/%{repo_orgname}/%{repo_name}/
+BuildArch: noarch
+
+Requires: ansible
+
+%description
+This package installs the satellite_receptor_installer Ansibile role.
```

Comment:
It'd be nice to know what the role does ;)

Reply:
ok, will add a bit more description of the role